src/login/SignInPage.js

```diff
@@ -66,18 +84,18 @@ export default function SignInPage({
           throw new Error(res.message)
         }
         setToStorage('pingu-session', res.token)
-        setCurrentUser(data)
+        setIsLoggedIn(true)
         form.reset()
         history.push('/users')
       })
       .catch(err => {
-        if (err.message === 'Error: email must not be blank') {
+        if (err.message === 'Email must not be blank') {
```

Comment:
Sind das Kommentare, die in der Console ausgegeben werden? Sonst vllt auf Deutsch, der Rest der App ist ja auch Deutsch, oder?